Translation corrections (#213)
07th-mod/higurashi-console-arcs: Update/tsuk_016.txt

//c900.r一番手前にいた老婆に、そっと声をかけてみる。一心不乱に拝んでいるせいか、私の声は届いていないようだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "<color=#ff0000>一番手前にいた老婆に、そっと声をかけてみる。一心不乱に拝んでいるせいか、私の声は届いていないようだ。</color>",

> 			NULL, "<color=#ff0000>I tried to gently speak to the old man in the front. But he was so intently focused on his prayers that my voice didn't seem to reach his ears.</color>", Line_Normal);

Reviewer: ```suggestion
			NULL, "<color=#ff0000>I tried to gently speak to the old woman in the front. But she was so intently focused on her prayers that my voice didn't seem to reach her ears.</color>", Line_Normal);
```

//c900.rその隣の老婆に声をかけてみる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "<color=#ff0000>その隣の老婆に声をかけてみる。</color>",

> 			NULL, "<color=#ff0000>I then tried calling out to the old man next to him.</color>", Line_Normal);

Reviewer: ```suggestion
			NULL, "<color=#ff0000>I then tried calling out to the old woman next to her.</color>", Line_Normal);
```

	ClearMessage();

//c900.r老婆は顔を上げると、念仏を邪魔されたのが口惜しいのか、怒りと苛立ちに満ちた形相で振り返る。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "<color=#ff0000>老婆は顔を上げると、念仏を邪魔されたのが口惜しいのか、怒りと苛立ちに満ちた形相で振り返る。</color>",

> 			NULL, "<color=#ff0000>The old man looked back at me with frustration and rage for having interrupted his prayer.</color>", Line_Normal);

Reviewer: ```suggestion
			NULL, "<color=#ff0000>The old woman looked back at me with frustration and rage for having interrupted her prayer.</color>", Line_Normal);
```
